queue_job/migrations/13.0.1.2.0/noupdate_changes.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This file `noupdate_changes.xml` seems not necessary anymore and could be removed.

Reply:
Removed